src/modules/link-tooltip.coffee

```diff
@@ -82,6 +84,7 @@ class LinkTooltip extends Tooltip
         @textbox.setSelectionRange(0, url.length)
       )
     else
+      dom(@container).addClass('open')
```

Comment:
If .open is rendered from show(), there will be situations where `open` and `editing` will coexist in the DOM. Looking forwards, I can think of several possible reasons for why we might want them to exist only in isolation -- for instance, custom placement, custom styling of certain buttons, etc.